### PR TITLE
DDT-1136

### PR DESCRIPTION
**Pull request by FinnAtAsseco**

### CODE REVIEW

File: jit-service/src/main/java/org/dcsa/jit/mapping/TransportCallMapper.java [hunk 1]
```diff
@@ -18,8 +18,8 @@
 public abstract class TransportCallMapper {
   @Mappings(
       value = {
-        @Mapping(target = "importVoyageNumber", source = "importVoyage.carrierVoyageNumber"),
-        @Mapping(target = "exportVoyageNumber", source = "exportVoyage.carrierVoyageNumber"),
+        @Mapping(target = "carrierImportVoyageNumber", source = "importVoyage.carrierVoyageNumber"),
+        @Mapping(target = "carrierExportVoyageNumber", source = "exportVoyage.carrierVoyageNumber"),
```
[the remark this answers is not in the document]
Comment (FinnAtAsseco) [reply jit-service/src/main/java/org/dcsa/jit/mapping/TransportCallMapper.java:22]: I thought the idea was to allow jit 1.1 input, but always return jit 1.2

[the remark this answers is not in the document]
Comment (FinnAtAsseco) [reply jit-service/src/main/java/org/dcsa/jit/mapping/TransportCallMapper.java:22]: Fixed, will now map both

File: jit-integration-tests/src/main/java/org/dcsa/jit/itests/v1/PostTimestampsIT.java [hunk 2]
```diff
@@ -710,8 +736,8 @@ public void testLogicForVoyageNumberFields() {
       .assertThat()
       .statusCode(400);
 
-    // remove exportVoyageNumber to test that when(exportVoyageNumber & importVoyageNumber) are not given together.
-    map.remove("exportVoyageNumber");
+    // remove carrierExportVoyageNumber to test that when(carrierExportVoyageNumber & carrierImportVoyageNumber) are not given together.
+    map.remove("carrierExportVoyageNumber");
```
Comment (Abdiiir) [inline jit-integration-tests/src/main/java/org/dcsa/jit/itests/v1/PostTimestampsIT.java:740]: This is still a valid timestamp1_1 (line 727), so nothing is being removed in the map. 

I think we should keep this test, as part of backwards comp for JIT 1.1

File: jit-integration-tests/src/main/java/org/dcsa/jit/itests/v1/PostTimestampsIT.java [hunk 3]
```diff
@@ -700,7 +726,7 @@ public void testMandatoryEventDateTimeFieldFalseFormat() {
   public void testLogicForVoyageNumberFields() {
```
Comment (Abdiiir) [inline jit-integration-tests/src/main/java/org/dcsa/jit/itests/v1/PostTimestampsIT.java:726]: New test should be added, instead of changing this one.

**Review by Abdiiir — CHANGES_REQUESTED** [review]

File: jit-transfer-obj/src/main/java/org/dcsa/jit/transferobjects/TimestampTO.java [hunk 4]
```diff
@@ -18,7 +18,7 @@
 public record TimestampTO(
   @NotNull PartyTO publisher,
   @NotNull PublisherRole publisherRole,
-  @Deprecated @ValidVesselIMONumber String vesselIMONumber, // Deprecated in JIT 1.2
+  @Deprecated @NotNull @ValidVesselIMONumber String vesselIMONumber, // Deprecated in JIT 1.2
```
Comment (Abdiiir) [inline jit-transfer-obj/src/main/java/org/dcsa/jit/transferobjects/TimestampTO.java:21]: No need since, @ValidVesselIMONumber also checks for null.

**Review by Abdiiir — DISMISSED** [review]

**Review by douradofrazer — APPROVED** [review]

**Pull request merged**